global/scripts/semgrep/run.sh

```diff
@@ -11,23 +11,46 @@ fileName="$CONTAINER_PATH/$REPORT_PATH/semgrep.json"
 
 defaultFile="$SCRIPTS_DIR/global/scripts/semgrep/.semgrepignore"
 cp "$defaultFile" .
-
 if [ -f ".semgrep.yaml" ]; then
-  docker run \
-  -v "$(pwd):$CONTAINER_PATH" \
-  --workdir "$CONTAINER_PATH" \
-  returntocorp/semgrep:latest \
-  semgrep \
-  --metrics=off \
-  --config "p/$1" \
-  --config "p/docker" \
-  --config "p/dockerfile" \
-  --config "p/secrets" \
-  --config "p/owasp-top-ten" \
-  --config "p/r2c-best-practices" \
-  --config ".semgrep.yaml" \
-  --enable-version-check --force-color \
-  --error --json --output "$fileName" || EXIT_CODE=$?
+  if [ -f ".semgrepcustom" ]; then
+    SEMGREP_CUSTOM=""
+    IFS=';'
```

Comment:
Could you test if it works in SH as well? I know you tested using BASH. But I'm afraid it won't work in SH inside GitLab DIND for example.

Reply:
Test works fine.
![sh test](https://github.com/rios0rios0/pipelines/assets/83853144/ad382aef-d32a-41b9-bd10-9283b50bf260)